editor/dependency_editor.cpp

```diff
@@ -350,7 +354,7 @@ DependencyEditorOwners::DependencyEditorOwners(EditorNode *p_editor) {
 
 	owners = memnew(ItemList);
 	owners->set_select_mode(ItemList::SELECT_SINGLE);
-	owners->connect("item_rmb_selected", callable_mp(this, &DependencyEditorOwners::_list_rmb_select));
+	owners->connect("item_clicked", callable_mp(this, &DependencyEditorOwners::_list_rmb_select));
```

Comment:
I suggest renaming the editor callbacks to also match the new names, so that it's not confusing. So e.g. here `_list_item_clicked`.

Reply:
BTW shouldn't this be connected to `item_selected` instead of `item_clicked`?

Reply:
Also changed `_item_list_rmb_clicked` to `_item_list_rmb_empty_clicked`